### PR TITLE
AX_ASM_INLINE: accept asm also

### PR DESCRIPTION
**Pull request by rurban**

### CODE REVIEW

File: m4/ax_asm_inline.m4 [hunk 1]
```diff
@@ -23,13 +23,13 @@
...
 
-#serial 4
+#serial 5
 
 AC_DEFUN([AX_ASM_INLINE], [
   AC_LANG_PUSH([C])
   AC_MSG_CHECKING(for inline assembly style)
   AC_CACHE_VAL(ac_cv_asm_inline, [
-    ax_asm_inline_keywords="__asm__ __asm none"
+    ax_asm_inline_keywords="__asm__ __asm asm none"
```
Comment (peti) [inline m4/ax_asm_inline.m4]: Why is this necessary? Would you mind what use case this change solves?

**Comment by rurban** [comment]
`asm` is in Annex J of the C Standard: `J.5.10 The asm keyword`
I haven't checked all esoteric compilers, but most code now uses just `asm`, not `__asm__`
